examples/guestbook/README.md

```diff
@@ -152,12 +167,12 @@ Use the file `examples/guestbook/redis-slave-controller.json`:
 to create the replication controller by running:
 
 ```shell
-$ cluster/kubectl.sh create -f examples/guestbook/redis-slave-controller.json
-redis-slave-controller
+$ kubectl create -f examples/guestbook/redis-slave-controller.json
+redisSlaveController
```

Comment:
Why did the output change (from redis-slave-controller to redisSlaveController) ?